test/parallel/test-tls-server-verify.js

```diff
@@ -262,10 +268,13 @@ function runTest(testIndex) {
 
   var renegotiated = false;
   var server = tls.Server(serverOptions, function handleConnection(c) {
+    c.on('error', function(e) {
+      // ignore errors
+    });
```

Comment:
Why is it necessary to swallow errors?  It looks suspect but if there is a good reason, can you add a comment explaining it?